synapse/rest/admin/users.py

```diff
@@ -182,6 +183,20 @@ def _filter(a: attr.Attribute) -> bool:
 
         return HTTPStatus.OK, ret
 
+    def _parse_parameter_deactivated(
+        self, request: SynapseRequest
+    ) -> Union[bool, None]:
+        return None if parse_boolean(request, "deactivated") else False
```

Comment:
Just to document this function, as I needed a moment to think about it personally:

```suggestion
        """
        Return None (no filtering) if `deactivated` is `true`, otherwise return `False`
        (exclude deactivated users from the results).
        """
        return None if parse_boolean(request, "deactivated") else False
```

The other implementation of `_parse_parameter_deactivated` is simpler and probably doesn't need a docstring.